scripts/ci-build.sh

```diff
@@ -37,6 +38,4 @@ else
 fi
 
 export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
-$EXEC pkg-config --cflags deltachat
-$EXEC pkg-config --libs deltachat
-$EXEC npm install;
+$EXEC npm install --dc-system-lib=$SYS_DC_CORE;
```

Comment:
Where is this `--dc-system-lib` going?

Reply:
It's handled by `scripts/npm-install.js` to either invoke `npm run rebuild-all` or invoke `node-gyp rebuild -- -Dsystem_dc_core=true` when calling `npm install`.

Reply:
@flub Nevermind, found it :)